### ISSUE TITLE
Chat theme / layout update

### ISSUE THREAD

**Issue by jjlankinen** [otavanopisto/muikku#5208]
Currently chat theme has few accessibility issues regarding contrast etc. and we should address these.

We also need to design chat layout somewhat differently when we start to add buddy features and add place to display ongoing private chats etc. Current implementation does not take into account any of these features yet.

Room listing categories should be foldable with accordion effect as users could have multiple chat rooms available to them and the list might be overly long to be usable.

Presence select should be relocated and redesigned as form element is unnecessary and it just takes too much space which is limited by default.
